<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Hakans-Air" name="HubSpot Functional Regression Test -- firefox" tests="9" failures="5" timestamp="24 Jul 2020 00:16:46 GMT" time="192.606" errors="0">
  <testcase name="verifyHomePageTitle" time="5.034" classname="com.qa.hubspot.tests.HomePageTest"/>
  <testcase name="verifyPageTitleTest" time="4.604" classname="com.qa.hubspot.tests.LoginPageTest"/>
  <testcase name="verifySignUpLink" time="6.002" classname="com.qa.hubspot.tests.LoginPageTest"/>
  <testcase name="verifyHomePageHeaderTest" time="20.275" classname="com.qa.hubspot.tests.HomePageTest">
    <failure type="java.lang.AssertionError" message="expected [Thanks for choosing HubSpot] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [Thanks for choosing HubSpot] but found [null]
	at com.qa.hubspot.tests.HomePageTest.verifyHomePageHeaderTest(HomePageTest.java:51)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- verifyHomePageHeaderTest -->
  <testcase name="login_invalidTestCase" time="5.775" classname="com.qa.hubspot.tests.LoginPageTest"/>
  <testcase name="login_invalidTestCase" time="25.84" classname="com.qa.hubspot.tests.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.qa.hubspot.tests.LoginPageTest.login_invalidTestCase(LoginPageTest.java:117)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- login_invalidTestCase -->
  <testcase name="login_invalidTestCase" time="26.133" classname="com.qa.hubspot.tests.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.qa.hubspot.tests.LoginPageTest.login_invalidTestCase(LoginPageTest.java:117)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- login_invalidTestCase -->
  <testcase name="login_invalidTestCase" time="26.966" classname="com.qa.hubspot.tests.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.qa.hubspot.tests.LoginPageTest.login_invalidTestCase(LoginPageTest.java:117)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- login_invalidTestCase -->
  <testcase name="login_invalidTestCase" time="25.633" classname="com.qa.hubspot.tests.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.qa.hubspot.tests.LoginPageTest.login_invalidTestCase(LoginPageTest.java:117)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- login_invalidTestCase -->
</testsuite> <!-- HubSpot Functional Regression Test - firefox -->
